packages/json-rpc-engine/package.json

```diff
@@ -47,7 +47,7 @@
     "@metamask/utils": "^8.3.0"
   },
   "devDependencies": {
-    "@lavamoat/allow-scripts": "^2.3.1",
```

Comment:
Oh weird — these aren't supposed to be here. I'll make a note to remove this in another PR.